lib/Controller/FileController.php

```diff
@@ -56,6 +63,13 @@ public function getPersonsFromPath(string $fullpath) {
 		$fileId = $userFolder->get($fullpath)->getId();
 
 		$resp = array();
+
+		$image = $this->imageMapper->findFromFile($this->userId, $fileId);
+		$resp['image_id'] = $image ? $image->getId() : 0;
+		$resp['is_processed'] = $image ? $image->getIsProcessed() : 0;
+		$resp['error'] = $image ? $image->getError() : null;
+
+		$aPersons = array();
```

Comment:
you don't need this variable. You can just use `$resp['persons']`?

Reply:
Something like that?
I have to force the creation of the *$resp['persons']*  to verify that it is empty..

Reply:
exactly what I had in mind! thanks!